science/volk-gnss-sdr/Portfile

```diff
@@ -38,11 +38,11 @@ subport volk-gnss-sdr-devel {
         This port is kept up with the VOLK-GNSS-SDR GIT next branch, which is typically updated daily to weekly. This version of VOLK-GNSS-SDR generally contains fixes and new features that will be incorporated in an upcoming release.
 
     name            volk-gnss-sdr-devel
```

Comment:
I *think* you should not set the `name` here as that is causing the `lint` error

Reply:
Removed.

Reply:
it's still there and thus is the `port lint` warning about it

Reply:
Now it is removed from both Portfiles :-)